components/docs.jsx

```diff
@@ -0,0 +1,124 @@
+            </p>
+            <p>
+              “I know,” the old man said. “It is quite normal.”
+            </p>
+            <p>
+              “He hasn’t much faith.”
+            </p>
+            <p>
+              “No,” the old man said. “But we have. Haven’t we?”
+            </p>
+            <p>
+              “Yes,” the boy said. “Can I offer you a beer on the Terrace and then we’ll take the stuff home.”
+            </p>
+            <p>
+              “Why not?” the old man said. “Between fishermen.”
+            </p>
+            <p> They sat on the Terrace and many of the fishermen made fun of the old man and he was not angry. Others, of the older fishermen, looked at him and were sad. But they did not show it and they spoke politely about the current and the depths they had drifted their lines at and the steady good weather and of what they had seen. The successful fishermen of that day were already in and had butchered their marlin out and carried them laid full length across two planks, with two men staggering at the end of each plank, to the fish house where they waited for the ice truck to carry them to the market in Havana. Those who had caught sharks had taken them to the shark factory on the other side of the cove where they were hoisted on a block and tackle, their livers removed, their fins cut off and their hides skinned out and their flesh cut into strips for salting.
+            </p>
+            <p> When the wind was in the east a smell came across the harbour from the shark factory; but today there was only the faint edge of the odour because the wind had backed into the north and then dropped off and it was pleasant and sunny on the Terrace.
+            </p>
+            <p> “Santiago,” the boy said. </p>
+            <p> “Yes,” the old man said. He was holding his glass and thinking of many years ago. </p>
+            <p> “Can I go out to get sardines for you for tomorrow?” </p>
+            <p> “No. Go and play baseball. I can still row and Rogelio will throw the net.” </p>
+            <p> “I would like to go. If I cannot fish with you. I would like to serve in some way.” </p>
+            <p>“You bought me a beer,” the old man said. “You are already a man.”</p>
+            <p> “How old was I when you first took me in a boat?” </p>
+            <p> “Five and you nearly were killed when I brought the fish in too green and he nearly tore the boat to pieces. Can you remember?”
+            </p>
+            <p>“I can remember the tail slapping and banging and the thwart breaking and the noise of the clubbing. I can remember you throwing me into the bow where the wet coiled lines were and feeling the whole boat shiver and the noise of you clubbing him like chopping a tree down and the sweet blood smell all over me.”
+            </p>
+          </section>
+        </main>
+        <Footer/>
+        <Style rules={theme}/>
+      </div>
+    )
+  }
+}
+
+if (typeof document !== "undefined") {
```

Comment:
I think we're better off *not* having the component render itself now that we've got a router--we can just export it as-is. :)